setup.cfg

```diff
@@ -43,12 +42,12 @@ install_requires =
 numpy = numpy
 dev =
     coverage
-    hypothesis~=5.49.0
+    hypothesis~=6.56.0
     mock
     pytest-cov
     pytest-mock
     pytest-xdist
-    pytest~=6.2.5
+    pytest~=7.1.3
```

Comment:
`pytest 7.2.0` was released on Oct 25. We can just bump this to 7.2.0, still working fine on my side.